src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,115 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+// import axios from 'axios';
```

Comment:
- Please, let's remove this commented-out piece of code to keep our codebase clean and maintainable.